package/src/types/v2-mmr.ts

```diff
@@ -0,0 +1,38 @@
+import { Rank, Season } from "./general.js"
```

Comment:
Why are you importing javascript files here?

Also this files doesn't even exist.